data/hai/hai reveal 1 intro.txt

```diff
@@ -63,7 +63,7 @@ mission "Event: Pug Media Coverage"
 			`As you walk through the spaceport, you can't help but notice that coverage on the events of the Pug invasion still continues to dominate headlines, even though it's been weeks now.`
 			`	Looking around, you see that experts and officials have given way to talking heads and theorists. Some of them are saying what you also suspect: that the Pug invasion was a ploy to cause humanity to band together, learn to resolve differences without conflict, and ensure the sorts of extremism that led to the events precipitating the war would not be tolerated.`
 			`	But others...`
-			`	One particular paranoid fringe-world politician dominates the air on the channel you're watching here. Councilor Maduena Kabiru from Bounty demands that the edges of human space be better understood, that threats and even potential threats must clearly exist beyond what is already known, and that it is the responsibility of the government to find out and inform the people. While one could definitely argue in favor of his point, it doesn't carry much weight coming from a councilor whose system is a mere two jumps from a brand new Navy base.`
+			`	One particular paranoid fringe-world politician dominates the air on the channel you're watching here. Senator Maduena Kabiru from Bounty demands that the edges of human space be better understood, that threats and even potential threats must clearly exist beyond what is already known, and that it is the responsibility of the government to find out and inform the people. While one could definitely argue in favor of his point, it doesn't carry much weight coming from a sentator whose system is a mere two jumps from a brand new Navy base.`
```

Comment:
Bounty is a Republic world, and the Republic has councilors, so isn't this correct as is?

Reply:
Some time back I was in a conversation where the conclusion was that Councilor referred to the FW Council members - which is disbanded at the end of the war when the FW Parliament takes over.
As a consequence of that the use of Senator became common in a few conversations I'd had about Republic politics (with an assumption of bicameralism).
More recently on the other PR LordInsane pointed out that there is indeed original text referring to Katya as a Councilor in her capacity as an elected official in the Republic... which frankly just makes for fun extra confusion for sure.

My preference is for this to be Senator eventually, but when I pushed for the other PR to be merged I had decided it shouldn't matter *for now*.

In the very near future I would like to put together a little outline of what all the different governments could look like; their structures, positions, titles, roles, etc. and then go through and make sure it's all standardised accordingly without reusing terms as much as possible (and specifically without reusing terms between the Republic and the FW).

Reply:
> Some time back I was in a conversation where the conclusion was that Councilor referred to the members of the FW Council... More recently on the other PR LordInsane pointed out that there is indeed original text referring to Katya as a Councilor in her capacity as an elected official in the Republic... which frankly just makes for fun extra confusion for sure.

Are the members of the FW Council ever referred to as councilors in their capacity as a member of the Council? From what I can find, only Katya is ever referred to as a councilor as an honorary title from when she was in Parliament.

> (with an assumption of bicameralism)

From what I can tell, that's a bad assumption. One of the reasons the Free Worlds secedes is because the South, Rim, and Dirt Belt lack the political power in the Republic Parliament to enact the changes they want from within the existing system, and part of that is because representation in Parliament is passed on population. 

From the historical timeline:
> 2760: The paradise worlds join the Republic, but with concessions that include *a restructured Parliament where very populous worlds have greater representation*, which gives the paradise planets considerable power.
> 3000: The galaxy is beginning to slip into decadence, with megacorporations like the Syndicate once again gaining power. The Republic has made several unpopular decisions, including expensive licenses that make it hard to make a start as an independent pilot, and interstellar trade tariffs that weigh most heavily on low-value commodities such as food and clothing. This, *combined with the Parliament structure that gives almost no voice to sparsely populated worlds*, has led to an increase of anti-Republic sentiment.

If Parliament was bicameral, then would both chambers be based on population? I've always taken these references to suggest that Parliament is unicameral, with a single population based chamber. Two population based chambers seems redundant (although technically not impossible, particularly if the means of using population to determine representation differed between the two), and if there were a second chamber that wasn't population based then I'd imagine we'd hear about it explicitly.

Also notable is how the times that you do enter Parliament, there's no distinction between chambers that you're in. You're just "in Parliament." There is an instance of "Parilament chambers" plural, but that changes to "chamber" singular within the same conversation. Other instances use "chamber" singular.

> free worlds 3 checkmate.txt: Finally you are invited into the Parliament chambers, and Douglas Xao, the Speaker, says... Alondo says, "That's right. Tungsten, to be exact..." As he speaks, the Parliament chamber gradually grows very quiet.
> free worlds 3 reconciliation.txt: Parliament is eager now to meet with you and to receive whatever assistance you may be able to offer. As Alondo stands in front of the chamber asking the members of Parliament for more information

Parliament being a unicameral, population based legislature would also mirror the Free Worlds Senate seemingly having a unicameral, one-senator-per-planet/system legislature.

> My preference is for this to be Senator eventually, but when I pushed for the other PR to be merged I had decided it shouldn't matter for now.

My preference is the opposite. If we want to avoid duplication of terms, the Free Worlds already has a Senate with senators. As such, I don't think it'd make much sense for the members of the Republic Parliament to also be called senators. As has been noted, they're already referred to as councilors, so let's stick with the language we already have.

Reply:
I changed Senators to Councilors at first, and removed the change after it was questioned. I agree with all of your points, but I won't change the dialogue now until we can hash something out. At the moment the official positions in the Republic seem to be MP and Councilor, and I don't know whether there is a difference or not.

It seems like members of the Free Worlds' Council are never actually called councilors, and Katya is always called Councilor because of her previous position. In the same sentence, Danforth says "Miss Freya" then "Councilor Reynolds", and nobody ever calls Tomek, Jean-Jacques, or Alondo councilors. Freya just says "Council members".

Reply:
Ok I do have some assumptions to challenge while I'm here before I add a proposal.

> Are the members of the FW Council ever referred to as councilors in their capacity as a member of the Council? From what I can find, only Katya is ever referred to as a councilor as an honorary title from when she was in Parliament.

Perhaps not - but people on Discord quite commonly seem to have made that assumption in prior discussions. Therefore I think this is worth fixing one way or another.

> > (with an assumption of bicameralism)
> 
> From what I can tell, that's a bad assumption. One of the reasons the Free Worlds secedes is because the South, Rim, and Dirt Belt lack the political power in the Republic Parliament to enact the changes they want from within the existing system, and part of that is because representation in Parliament is based on population.

It mostly comes from the simple fact that the Republic managed to be relatively stable for ~210 years. That sort of stability is not typical of unicameral parliaments, but is of bicameral parliaments.
The common political understanding is that, "Bicameral legislature is inherently more stable, more restrained in its actions, and therefore more likely to preserve a desirable steadiness and reliability in the law."

> If Parliament was bicameral, then would both chambers be based on population?

It could very easily be. The US may not afford representation to its territories, but in Australia (for example) the representative house is purely population based, and the senate is threshold population based. Low population areas which were territories get 2 senators, and full states which had substantial population/industry/etc. have 12. That is a real-world example of a bicameral system where populous regions built in advantages for themselves over any current or potential future less populous ones.
(The fact that the territories have caught up to one state which has lagged is an argument for better representation of the territories, which is actually an almost perfect mirror of the FW desire for better representation in the game.)

> Also notable is how the times that you do enter Parliament, there's no distinction between chambers that you're in. You're just "in Parliament." There is an instance of "Parilament chambers" plural, but that changes to "chamber" singular within the same conversation. Other instances use "chamber" singular.

In bicameral systems there are typically certain decisions which can only be *initiated* in one particular house. (To use Australia as an example again; anything at all that is remotely related to the level of taxation can *only* be put forward in the house of representatives.) The house which has sole jurisdiction over some subset is typically the one referred to as "Parliament" in singular.
In some other countries with bicameral systems the ability to declare war is an act reserved for the "parliament" and is also tied to the one house in the same way.

> Parliament being a unicameral, population based legislature would also mirror the Free Worlds Senate seemingly having a unicameral, one-(maybe-two)-senator(s)-per-planet/system legislature.
> 
> > free worlds 1 start.txt: Several hours pass. Finally, you are led into the Senate chamber, where several dozen senators sit, each with a name card identifying the world they represent.

But why, with the history of all the governments ever as examples, and a clear desire to do something in a way they perceive to be better than the Republic, would the FW deliberately mirror them?

> > My preference is for this to be Senator eventually, but when I pushed for the other PR to be merged I had decided it shouldn't matter for now.
> 
> My preference is the opposite. If we want to avoid duplication of terms, the Free Worlds already has a Senate with senators. As such, I don't think it'd make much sense for the members of the Republic Parliament to also be called senators. As has been noted, they're already referred to as councilors, so let's stick with the language we already have.

This I can get, not having senators in the Republic if we're explicitly keeping a unicameral senate (which *does* actually make sense for the FW) seems a good choice - but "councilor" is still a terrible title that we should seek alternatives for. The world over it's almost exclusively used for one of two things:
- a low-level elected position within a sub community like a single city or even a single neighbourhood.
- a very high-level *appointed* position with powers above that of the elected representatives (usually a position that upgrades someone who is already an elected representative in democracies)

Currently in the game it's being used for a high-level elected representative with no particular special responsibilities...
It's a bad term for what it's applied to.

We could variously spread out the use of: representative, legislator, senator, congresswoman/man/person, minister, assemblywoman/man/person, governor, stateswoman/man/person, delegate, advocate, designate...
Most of these would be better than councillor for her specifically as an elected rep from a FW world.
All of these could be better than councillor for some combination of worlds depending on how they appoint whichever people it is that represent them in the Republic Parliament.

Reply:
> It mostly comes from the simple fact that the Republic managed to be relatively stable for ~210 years. That sort of stability is not typical of unicameral parliaments, but is of bicameral parliaments.

You can have a bad system and still have glory days, like the US in the 40s-70s. And obviously, right now the Republic is not doing so hot. But, this is governance in a thousand years' time. Everything could be done with AI for what we know. It doesn't look like it, though. 

I agree that if Katya is a member of Parliament, Councilor is a bad word for it. I think that the Free Worlds, at the very least, would try to differentiate themselves from the Republic, because that's their whole shtick. That means no Senators in the Republic.

An aside... I find it very funny that the "Republic" runs on a "Parliament" with MPs and "Senators" (or Councilors?), headed by a "President". The only thing consistent about the Republic's politics is the inconsistency in naming their political concepts. Well, I suppose you guys across the ditch have MPs in a Senate, under a commonwealth... 

How about "Representative"?

Reply:
President isn't really an issue as a title. It's far from unknown for "President" to be part of the official title of the head of government in a parliamentary or semi-parliamentary system - for example, Spain has the President of the Government, Italy has the President of the Council of Ministers, and the EU has the President of the European Commission.

Reply:
I would rather have this PR done just to fix the typos, not to change the Republic's policital positions, which will probably affect other things down the line. Unless there is something else I need to change, I don't see anything else that's blocking it.